opm/grid/CpGrid.hpp

```diff
@@ -262,10 +262,10 @@ namespace Dune
         /// \param turn_normals if true, all normals will be turned. This is intended for handling inputs with wrong orientations.
         /// \param clip_z if true, the grid will be clipped so that the top and bottom will be planar.
         /// \param poreVolume pore volumes for use in MINPV processing, if asked for in deck
-        void processEclipseFormat(const Opm::EclipseGrid* ecl_grid, bool periodic_extension, bool turn_normals = false, bool clip_z = false,
-                                  const std::vector<double>& poreVolume = std::vector<double>(),
-                                  const Opm::NNC& = Opm::NNC(),
-                                  const std::unordered_map<size_t, double>& aquifer_cell_volumes = std::unordered_map<size_t, double>());
+        // TODO: add description of the arguments
+        void processEclipseFormat(Opm::EclipseState* ecl_state,
```

Comment:
One thing unfortunate due to the change here is that, originally the main input is `EclipseGrid`, but now the main input `EclipseGrid` becomes something hidden in the `EclipseState`. 

Maybe we still can keep `EclipseGrid` as the main input, but also has `EclipseState` there for extra information. But we can not guarantee the `EclipseGrid` is the grid from `EclipseState`.

Reply:
I will recovered to be the main argument is the `EclipseGrid`. `EclipseState` will be secondary argument to provide other information. When there is only `EclipseState` provided, we use the `EclipseGrid` belonging to `EclipseState`.